packages/create-react-native-library/src/utils/generateExampleApp.ts

```diff
@@ -65,17 +65,22 @@ export default async function generateExampleApp({
   const directory = path.join(dest, 'example');
   const args =
     type === 'native'
-      ? // `npx react-native init <projectName> --directory example --skip-install`
+      ? // `npx --package react-native-test-app@latest init --name ${projectName}Example --destination example --version ${reactNativeVersion}`
         [
-          'react-native@latest',
+          '--package',
+          `react-native-test-app@latest`,
           'init',
+          '--name',
           `${projectName}Example`,
-          '--directory',
+          `--destination`,
           directory,
-          '--version',
-          reactNativeVersion,
-          '--skip-install',
-          '--npm',
+          ...(reactNativeVersion !== 'latest'
+            ? ['--version', reactNativeVersion]
+            : []),
+          '--platform',
+          'ios',
+          '--platform',
+          'android',
```

Comment:
Q: should we ask users which platforms they want to add to `example/` app?

Reply:
and also adding new platform for existing RNTA project is just a matter of [one command](https://github.com/microsoft/react-native-test-app/wiki/Updating#adding-platforms-to-an-existing-app).